test-clients/src/main/java/com/hedera/services/bdd/suites/schedule/ScheduleRecordSpecs.java

```diff
@@ -123,6 +136,49 @@ HapiApiSpec canonicalScheduleOpsHaveExpectedUsdFees() {
 				);
 	}
 
+	public HapiApiSpec canScheduleChunkedMessages() {
+		String ofGeneralInterest = "Scotch";
+		AtomicReference<TransactionID> initialTxnId = new AtomicReference<>();
+
+		return defaultHapiSpec("CanScheduleChunkedMessages")
+				.given(
+						cryptoCreate("payingSender").balance(A_HUNDRED_HBARS),
+						createTopic(ofGeneralInterest)
+				).when(
+						withOpContext((spec, opLog) -> {
+							var subOp = usableTxnIdNamed("begin").payerId("payingSender");
+							allRunFor(spec, subOp);
+							initialTxnId.set(spec.registry().getTxnId("begin"));
+						}),
+						sourcing(() -> scheduleCreate("firstChunk",
+								submitMessageTo(ofGeneralInterest)
+										.chunkInfo(
+												3,
+												1,
+												scheduledVersionOf(initialTxnId.get()))
+										.signedBy("payingSender")
+								)
+										.txnId("begin")
+										.signedBy("payingSender")
+										.inheritingScheduledSigs()
+						),
+						getTxnRecord("begin").scheduled().hasPriority(recordWith().status(SUCCESS))
+				).then(
+						scheduleCreate("secondChunk",
+								submitMessageTo(ofGeneralInterest)
+										.chunkInfo(3, 2, "payingSender")
+										.signedBy("payingSender")
+						)
+								.payingWith("payingSender")
+								.inheritingScheduledSigs(),
+						getTopicInfo(ofGeneralInterest).logged().hasSeqNo(2L)
```

Comment:
Is it possible this `getTopicInfo(...)` may be executed earlier than the one or two of the scheduled `submitMessateTo(...)` and cause the check of `hasSeqNo(2L)` be unstable?

Reply:
Since the preceding `scheduleCreate` is scheduling a transaction with all required signatures available, it will immediately trigger the `submitMessage`.

The `continuous-integration-gcp` job failed because I hadn't updated the _test-clients/src/main/resource/bootstrap.properties_ to add `ConsensusSubmitMessage` to the `scheduling.whitelist`...should pass now. 🤞